### ISSUE TITLE
User initializer doesn't set superuser and staff status

### ISSUE THREAD

**Issue by TomaszKrassowski** [netbox-community/netbox-docker#676]
### Current Behavior

YAML structure in the users.yaml initializer suggests that user will be created with staff and superuser fields set.
This doesn't happen in the 000_users.py file, as it only checks username, password and api_token.

### Expected Behavior

Superuser/staff user should be created if is_superuser or is_staff are set to true:
```
jdoe:
   first_name: John
   last_name: Doe
   api_token: 0123456789jdoe789abcdef01234567jdoe
   is_active: True
   is_superuser: True
   is_staff: True
   email: john.doe@example.com
```
the fix is rather simple, as create_user function accepts extra_fields, which are then later on passed to the user manager to create database entity.

### Docker Compose Version

1.29.2

### Docker Version

```text
Client:
 Cloud integration: 1.0.17
 Version:           20.10.8
 API version:       1.41
 Go version:        go1.16.6
 Git commit:        3967b7d
 Built:             Fri Jul 30 19:58:50 2021
 OS/Arch:           windows/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.8
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.6
  Git commit:       75249d8
  Built:            Fri Jul 30 19:52:10 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.4.9
  GitCommit:        e25210fe30a0a703442421b0f60afac609f950a3
 runc:
  Version:          1.0.1
  GitCommit:        v1.0.1-0-g4144b63
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

### The git Revision

58a1579

### The git Status

```text
On branch release
Your branch is up to date with 'origin/release'.

nothing to commit, working tree clean
```

### Startup Command

-

### NetBox Logs

```text
-
```

### Content of docker-compose.override.yml

```yaml
-
```

**Comment by ryanmerolle** [comment]
> the fix is rather simple, as create_user function accepts extra_fields, which are then later on passed to the user manager to create database entity.

Feel free to submit a PR.

**Comment by tobiasge** [comment]
Fixed by PR #729
